tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -974,6 +973,61 @@ public void SQLite_InstallAndLogMessage()
             }
         }
 
+        private void SetupSqliteConfigWithInvalidInstallCommand(string databaseName)
+        {
+            var nlogXmlConfig = @"
+            <nlog xmlns='http://www.nlog-project.org/schemas/NLog.xsd'
+                  xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' throwExceptions='false'>
+                <targets>
+                    <target name='database' xsi:type='Database' dbProvider='{0}' connectionstring='{1}' 
+                        commandText='insert into RethrowingInstallExceptionsTable (Message) values (@message);'>
+                        <parameter name='@message' layout='${{message}}' />
+                        <install-command ignoreFailures='false' text='THIS IS NOT VALID SQL;' />
+                    </target>
+                </targets>
+                <rules>
+                    <logger name='*' writeTo='database' />
+                </rules>
+            </nlog>";
+
+            // Use an in memory SQLite database
+            // See https://www.sqlite.org/inmemorydb.html
+            var connectionString = String.Format("FullUri=file:{0}?mode=memory;Version=3", databaseName);
```

Comment:
OK, this doesn't seem to be supported in the Mono version. I've changed it to `Uri=file::memory:` which looks like both Mono and .NET support.